prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+<a href="file:///C:/Users/Topper/Downloads/prep-html-links/index.html"> Click here to go to the home page</a>
```

Comment:
This won't work on somebody else's computer. You'll want to use a relative path back to the index file.

```html
<a href="../index.html"> ... </a>
```

Reply:
will <a href="prep-html-links/index.html"> ... </a> work here as well?

Reply:
will `<a href="prep-html-links/index.html"> ... </a>` work here as well?